index.html

```diff
@@ -0,0 +1,293 @@
+          that requests a mobile drivers license from the user.
+        </p>
+        <p>
+          A credential provider is responsible for presenting the user with a
+          list of [=digital credential/request protocol=] specific providers
+          (e.g., a list of mobile drivers license or applications that hold
+          driver's licenses).
+        </p>
+      </dd>
+      <dt>
+        <dfn for="digital identity">request protocol</dfn>
+      </dt>
+      <dd>
+        The request protocol is a standardized format/structure to [=request
+        digital identity credentials=]. For example, a request protocol could
+        be a "mDoc" for mobile drivers licenses as per [[?ISO-18013-5]].
+      </dd>
+    </dl>
+    <h2>
+      Scope
+    </h2>
+    <p>
+      The following is in is scope for the this specification:
+    </p>
+    <ul>
+      <li>Requesting a [=digital identity credentials=].
+      </li>
+    </ul>
+    <p>
+      The following is out of scope
+    </p>
+    <ul>
+      <li>Creating, updating, deleting, or otherwise managing [=digital
+      identity credentials=].
+      </li>
+    </ul>
+    <h2>
+      Extensions to the `CredentialsContainer` interface
+    </h2>
+    <pre class="idl">
+    partial interface CredentialsContainer {
+      Promise&lt;DigitalCredential&gt; requestIdentity(IdentityRequestOptions options);
+    };
+    </pre>
+    <h3>
+      The `requestIdentity()` method
+    </h3>
+    <p data-dfn-for="CredentialsContainer">
+      When the <dfn>requestIdentity()</dfn> method is called, the user agent
+      MUST run the [=request digital identity credentials=] steps.
+    </p>
+    <h2>
+      The `IdentityRequestOptions` dictionary
+    </h2>
+    <p>
+      The {{IdentityRequestOptions}} dictionary is used to specify options when
+      requesting a digital identity credential.
+    </p>
+    <pre class="idl">
+    dictionary IdentityRequestOptions {
+      AbortSignal signal;
+      required sequence&lt;IdentityRequestProvider&gt; providers;
+    };
+    </pre>
+    <p>
+      The <dfn data-dfn-for="IdentityRequestOptions">providers</dfn> member is
+      a sequence of [=digital credential/request protocol=] specific providers.
+    </p>
+    <h3>
+      The `signal` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="IdentityRequestOptions">signal</dfn> member is an
+      {{AbortSignal}} that allows the caller to abort an ongoing [=request
+      digital identity credentials=].
+    </p>
+    <h2>
+      The `IdentityRequestProvider` dictionary
+    </h2>
+    <p>
+      The {{IdentityRequestProvider}} dictionary is used to specify a
+      [=digital credential/request protocol=] specific provider, while also handling
+      cryptographic requirements.
+    </p>
+    <pre class="idl">
+    dictionary IdentityRequestProvider {
+      required DOMString protocol;
+      required ArrayBuffer request;
+      required CryptoKey publicKey;
```

Comment:
Can we remove this until we get some convergence on this:

https://github.com/WICG/identity-credential/issues/56

I, for one, am not convinced that we need to have `publicKey` here at all.

Reply:
I think it can be valuable to have the public key here, but in the interest of landing the initial spec in github, I am supportive of seeing it added in a separate PR after this one.

Reply:
> Can we remove this until we get some convergence on this:
> 
> #56
> 
> I, for one, am not convinced that we need to have `publicKey` here at all.

We discussed this in the  2023-12-18 call:

meeting notes: https://github.com/WICG/identity-credential/wiki/2023-12-18-Meeting-Notes#do-we-really-need-publickey-to-be-outside-of-the-request-property

Consensus on the following: ( copied from the issue) 

the public key property should be required and should also live at the top level (independent of protocol)
the value should be more structured than just a DOMString
maybe redundant with a key acquired via an out-of-band method, but cannot be contradictory
need to further refine HPKE, including crypto agility

Happy to re-discuss it if needed.

Reply:
@samuelgoto, can you live with this here for now? (we need to put the publicKey somewhere, so this seems like an ok start?)

I added a link to #49 below so we can continue to discuss.

Reply:
It is not clear to me what the browser would (and should) do with the `publicKey`. Chrome implemented a prototype of this API deliberately delegating the validation of signatures to verifiers and holders, and it is not clear to me what benefit we get by pulling this responsibility to the browser. 

I can be convinced that interoperability/fragmentation is something that I think the browser can help, but I think it would be premature to over-generalize this aspect at the moment, since we don't even know what "protocols" are, let alone force all of them to comply to a specific way to check for digital signatures.

Reply:
Resolving this, since you removed it in the last commit:

https://github.com/WICG/digital-credentials/pull/57#issuecomment-1911289914